utils/datoCMS/api.ts

```diff
@@ -0,0 +1,122 @@
+import _ from 'lodash'
+import { request } from '../datocmsClient'
+
+const datoCMSAllTaxonomies = async (catalogId: string, locale = 'en-US') => {
+  const query = `{
+    allCatalogs(filter:{id:{eq:"${catalogId}"}}) {
+      taxonomies {
+      name
+      label
+      taxons {
+        name
+        label
+        products{
+          name
+          description
+          reference
+          slug
+          images {
```

Comment:
in Dato you could use directly an image, rather than a link to a model. By default images have title/alt tags that can be filled and localized, so you should be good to go like this and also have exactly the same data structure as before :)